### PR TITLE
Handling pending applications

### PR DESCRIPTION
**Pull request by mariiu**
auth commented out for testing purposes

### CODE REVIEW

File: admin-frontend/src/utils.js [hunk 1]
```diff
@@ -4,7 +4,13 @@ import { dataProvider } from './providers/dataProvider';
 export const genderChoices = [
```
Comment (RikuRuokonen) [inline admin-frontend/src/utils.js:4]: Minor comment: These could be extracted to constants etc. file, where some constant data or variables are stored. Imo utils are generally utility functions. But doesn't really matter for now if it works.

File: backend/src/junior/junior.service.ts [hunk 2]
```diff
@@ -131,15 +136,24 @@ export class JuniorService {
...
         user.gender = details.gender;
         user.nickName = details.nickName;
+        user.status = details.status;
+        user.photoPermission = details.photoPermission;
         const errors = await validate(user);
         if (errors.length > 0) {
             throw new BadRequestException(errors);
         }
         await this.juniorRepo.save(user);
+        if (prevStatus === 'p' && details.status === 'a') {
```
Comment (RikuRuokonen) [inline backend/src/junior/junior.service.ts:152]: What this line exactly does?

File: frontend/src/components/RegistrationView/FormFields.tsx [hunk 3]
```diff
@@ -50,109 +22,51 @@ export const InputField: React.FC<FieldProps & InputProps> = ({
...
     field,
-    title,
+    data,
     form,
     ...props
   }) => {
     return (
         <SelectOption>
-            <Radio type="radio" id={title} 
+            <Radio type="radio" id={data.value}
```
Comment (RikuRuokonen) [inline frontend/src/components/RegistrationView/FormFields.tsx:49]: Is it good that id comes from value? I think it's changing value? Now if its changing value, and 2 fields value is same --> have same id --> same htmlFor --> clicking one fields label targets both of the fields, or otherway acts weird.

Comment (mariiu) [reply frontend/src/components/RegistrationView/FormFields.tsx:49]: it's a radio button, the values are fixed and different

File: backend/src/junior/junior.service.ts (hunk 2 above)
Comment (mariiu) [reply backend/src/junior/junior.service.ts:152]: p is pending and a is accepted. I guess I shouldn't have mapped them to one-letter variables as has been done for gender, I'll change it back to normal words then

File: admin-frontend/src/utils.js (hunk 1 above)
Comment (mariiu) [reply admin-frontend/src/utils.js:4]: yeah, that's how it was before me, I didn't even think to reorganise the structure...

File: frontend/src/components/RegistrationView/FormFields.tsx (hunk 3 above)
Comment (mariiu) [reply frontend/src/components/RegistrationView/FormFields.tsx:49]: I mean it's not a value for the whole field, it's a value of each radio option

File: admin-frontend/src/utils.js (hunk 1 above)
Comment (RikuRuokonen) [reply admin-frontend/src/utils.js:4]: Yeah I kind guessed that might be the case. Well then it's not your responsibility to fix it only, so just leave it like that :)

**Pull request merged**